app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
+    if page_title.empty?
```

Comment:
dùng toán tử 3 ngôi cho gọn code nha em
page_title.empty? ? base_title : "#{page_title} | #{base_title}"